src/main/java/org/opentripplanner/transit/raptor/rangeraptor/RangeRaptorWorker.java

```diff
@@ -228,24 +228,8 @@ private void transfersForRound() {
 
     /**
      * Create a trip search using {@link TripScheduleBoardSearch}.
-     * <p/>
-     * This is protected to allow reverse search to override and create a alight search instead.
      */
     private TripScheduleSearch<T> createTripSearch(RaptorTimeTable<T> timeTable) {
-        if(!inFirstIteration && roundTracker.isFirstRound()) {
-            // For the first round of every iteration(except the first) we restrict the first
-            // departure to happen within the time-window of the iteration. Another way to put this,
-            // is to say that we allow for the access path to be time-shifted to a later departure,
-            // but not past the previous iteration departure time. This save a bit of processing,
-            // but most importantly allow us to use the departure-time as a pareto criteria in
-            // time-table view. This is not valid for the first iteration, because we could jump on
-            // a bus, take it one stop and walk back and then wait to board a later trip - this kind
-            // of results would be rejected by earlier iterations, for all iterations except the
-            // first.
-            return calculator.createExactTripSearch(timeTable);
-        }
```

Comment:
I measured the performance impact of removing this and it is around 5% for my bench mark test. My test do not have very long search-windows so I expect it to be higher for the common long distance search we have in Norway.

Reply:
I thought about this an I want to keep the optimization, but instead of deleting the code could you just comment it out. I can fix it in a separate PR, I have an idea on how to fix this. If you want to do it here is my initial thoughts:

This method:
```
if(useExactTimes) {
    // ...
    return calculator.createExactTripSearch(timeTable);
}
```
Then we can use the lifecycle events to set the `useExactTimes` before each round start. Also when the access is added we can turn it _off_ if any of the access is time-dependent access.

Reply:
Thanks for taking a look! Instead of making the exact times search optional, I've modified it so that the previous iteration's access time are kept track of. This may then be reused for the exact times search as an explicit window in which to accept boardings. Instead of `[timeLimit, timeLimit + iterationStep]`, `[timeLimit, previousIterationTimeLimit]` is used.